node_modules/cacache/lib/entry-index.js

```diff
@@ -19,6 +19,12 @@ const hashToSegments = require('./util/hash-to-segments')
 const indexV = require('../package.json')['cache-version'].index
 const { moveFile } = require('@npmcli/fs')
 
+const pMap = require('p-map')
+const lsStreamOpts = {
```

Comment:
This should either be a single value, or the object itself should be passed to pMap.

Reply:
Thank you. Changed lsStreamOpts to a single value lsStreamConcurrency = 5.